### DISCUSSION TITLE
Potential project?

### DISCUSSION THREAD

**Discussion by Matherion** [rbind/yihui#1558] [General]
Dear Yihui,

apologies for not having fully RTFMed your page at https://yihui.org/en/about/#contact-me 😬 I also sent you messages via LinkedIn and Mastodon 😬

And, first: THANK YOU for {knitr}, {rmarkdown}, {bookdown}, {blogdown}, and the whole cosmos of awesome tools you gifted to humanity! 🙏🙏🙏

I was looking at your Sponsoring page, and I realized there might be a win-win type of opportunity here 🤩 (i.e. no danger, just opportunity 😬)

Szilvia Zorgo and me recently got a grant to, among other things, commission a small R package, {qualchemy}. It would basically import and export files in a specific XML standard. These are files for qualitative research, in [the REFI-QDA format](https://www.qdasoftware.org/). The aim is to 'free' qualitative research projects (we develop the Reproducible Open Coding Kit, the ROCK, a standard for qualitative research in the Markdown and YAML spirit, i.e. both human- and machine-readable). Would that seem like something that's fun and worthwhile of your time?

(You can contact me here in this discussion, I'll keep the tab open. But I don't use GitHub much (in favour of open source alternatives, [GitLab](https://gitlab.com/matherion) and [CodeBerg](https://codeberg.org/matherion)), so if I accidentily close this tab, you could resort to [@matherion@mastodon.nl](https://mastodon.nl/@matherion) 😬)

**Comment by yihui** [comment]
Dear Gjalt-Jorn,

Thank you very much for contacting me! I just briefly looked at the REFI-QDA specs. It seems the main skill required for this project is XML, right? If possible, I'd love to have a chat with you t learn more. I'll email you.

**Comment by Matherion** [comment]
Awesome! I'm not sure with which e-mail address I registered here - my regular work one is gjalt-jorn.peters@ou.nl. And yes, I think the three skills you need are 1) package development, 2) XML, and 3) 'data modeling', as in, thinking together about how to best represent the imported project in R given the aim of the package (a kind of swiss army knife of importing/exporting to qualitative data formats; in addition to the ROCK / {rock}, there are the Python-based QualCoder, the Shiny-based ReQual, and some others).

I'll very happily chat about this of course! I guess that'd be my late afternoon or evening and your morning, roughly. But yeah, let's do e-mail (horrible though e-mail is 🙂)!

**Comment by Matherion** [comment]
Oew, found your mail, awesome, I'll reply to that! 🙂
